### DISCUSSION TITLE
手机端打开网站卡顿

### DISCUSSION THREAD

**Discussion by xiamuqingyu** [solstice23/argon-theme#220] [General]
左侧栏，折叠区块打开等等，在手机上面打开很卡，多个手机都很卡。

**Comment by yzxoi** [comment]
本人测试都是很流畅的呀？

**Comment by xiamuqingyu** [comment]
谢谢大佬回复✧*｡٩(ˊᗜˋ*)و✧*｡，手机打开左侧栏有点卡顿
还有一个bug，左侧栏有多个二级菜单的时候，打开上一个，再点下一个就会错误
从上往下展开点击正常，从下往上展开点击错误

---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2021年6月13日(周日) 上午8:26
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [solstice23/argon-theme] 手机端打开网站卡顿 (#220)




 
本人测试都是很流畅的呀？

**Reply by yzxoi** [reply]
https://user-images.githubusercontent.com/46484956/121792145-f027e480-cc23-11eb-9009-34db4216979f.mp4

二级菜单我试试

**Reply by yzxoi** [reply]
https://user-images.githubusercontent.com/46484956/121792189-57de2f80-cc24-11eb-8058-0c05b86ab1ba.mp4

手机上测试也不卡

**Comment by xiamuqingyu** [comment]
好的，可能是浏览器问题，感谢✧*｡٩(ˊᗜˋ*)و✧*｡



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2021年6月13日(周日) 上午8:50
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [solstice23/argon-theme] 手机端打开网站卡顿 (#220)




 
https://user-images.githubusercontent.com/46484956/121792189-57de2f80-cc24-11eb-8058-0c05b86ab1ba.mp4
 
手机上测试也不卡

**Reply by yzxoi** [reply]
另外如果是二级菜单的话，第一级最好是自定义 HTML 然后 URL 啥都不填或者填 `javascript:void(0);`

**Comment by xiamuqingyu** [comment]
好的

---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2021年6月13日(周日) 上午9:00
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [solstice23/argon-theme] 手机端打开网站卡顿 (#220)




 
另外如果是二级菜单的话，第一级最好是自定义 HTML 然后 URL 啥都不填或者填 javascript:void(0);

**Comment by xiamuqingyu** [comment]
端午快乐🥰



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2021年6月13日(周日) 上午9:00
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [solstice23/argon-theme] 手机端打开网站卡顿 (#220)




 
另外如果是二级菜单的话，第一级最好是自定义 HTML 然后 URL 啥都不填或者填 javascript:void(0);
